projects/ngx-speculoos/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Helps writing Angular unit tests",
   "peerDependencies": {
     "@angular/core": "^7.0.2",
+    "@angular/platform-browser": "^7.0.2",
```

Comment:
`@angular/platform-browser` was not listed as a peer dep, but is used in `test-element-querier.ts`.